app/src/main/kotlin/com/wire/android/notification/PendingIntents.kt

```diff
@@ -134,22 +138,22 @@ fun fullScreenIncomingCallPendingIntent(context: Context, conversationId: String
 
     return PendingIntent.getActivity(
         context,
-        FULL_SCREEN_REQUEST_CODE,
+        getRequestCode(conversationId, FULL_SCREEN_REQUEST_CODE),
```

Comment:
This is needed, otherwise we were always getting the initial intent to start the activity.